java-checks/src/main/java/org/sonar/java/checks/SQLInjectionCheck.java

```diff
@@ -42,11 +42,31 @@
 @SqaleConstantRemediation("20min")
 public class SQLInjectionCheck extends AbstractInjectionChecker {
 
+  private static final MethodMatcher HIBERNATE_SESSION_CREATE_QUERY_MATCHER = MethodMatcher.create()
+    // method from the interface org.hibernate.SharedSessionContract, implemented by org.hibernate.Session
+    .callSite(TypeCriteria.subtypeOf("org.hibernate.Session"))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)